collect.py

```diff
@@ -7,6 +7,18 @@
 from selenium import webdriver
 from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
 
+from sys import platform as _platform
```

Comment:
This is quite ugly 😄 
Just import `sys` at the top and then use `sys.platform` when you need it.